### ISSUE TITLE
Exacerbate by exposure to ultraviolet radiation

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10634]
**Preferred term label:**

**Synonyms**
Exacerbate by exposure to sunlight
Aggravated by ultraviolet radiation exposure

**Definition (free text, please give PubMed ID)**
PMID: 30137841 Darier disease is exacerbated by sunlight, heat, infections, and friction.

PMID: 26945535 High temperatures, high humidity, excessive sweating, pregnancy and delivery, surgery, **exposure to UV rays** and mechanical irritation are known to be exacerbating factors for DD.

PMID: 37545821 The first clinical symptoms of Darier disease usually appear in adolescence or early adulthood and may be associated with pruritus. Different factors are thought to exacerbate keratosis follicularis. Some of them include exposure to ultraviolet radiation, mechanical injuries, surgery procedures, humidity, heat, stress, menstruation, pregnancy and childbirth.

**Parent term (use hpo.jax.org/app)**
Aggravated by HP:0025285

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:218 DARIER DISEASE

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001075
- label: Exacerbated by exposure to sunlight
- parents: [Aggravated by exposure to medication (HP:4000176)](https://hpo.jax.org/app/browse/term/HP:4000176)
- pmid: 
- def: Applied to a sign or symptom that is worsened by exposure to sunlight.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10634
- synonyms: Aggravated by ultraviolet radiation exposure(NONE)
